starter-code/index.html

```diff
@@ -0,0 +1,253 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+        integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <link rel="stylesheet" href="../starter-code/style.css">
+    <title>Lab Bootstrap Cloning Revera</title>
+</head>
+
+<body>
+    <heading>
+        <div class="row">
+            <div id="home page" class="container-fluid">
```

Comment:
Does this div, including its classes, belong in the heading?
I reckon it could be the first element nested in the body tag.

Reply:
Applying the "container-fluid" class will also fix the width issues with the footer tag.